### DISCUSSION TITLE
SB coKreat working group creation

### DISCUSSION THREAD

**Discussion by mmanimegalai** [Sunbird-coKreat/Community#78] [General Discussions]
All,

We are forming a working group for SB coKreat. All the people who are actively contributing to this building block or using it, can nominate themselves to be part of this group. Whoever would like to nominate, please reply to this thread. We request that there should not be more than three people from each organization, who can be nominated, to be part of this working group. The nominated individuals should have made significant contributions to coKreat or have used coKreat and are SMEs in coKreat.

Responsibilities of working Group members:

1. Engage actively in the discussion forums and with the community by providing feedback to new ideas, technical or process related clarifications, helping plan events, etc.
2. Accept/Reject New Requirements, Tech Design, PRs
3. Can be sponsors for New features
4. Define community practices
5. Enable ease of adoption
6. Define Building Block roadmap
7. Advocate evolution of capabilities that increase the value-offering of the building block. 
8. Foster community growth and facilitate vibrant community.

Note: The Working Group will meet once in two weeks to connect and discuss about new ideas, open issues and take any decisions required or define action items.

The date and link for the first Working Group meeting will be shared soon 

@sanyapawah @aayushkedia95 @ChakshuGautam @abhay-samagra
@coolbung @parthlawate
@kameshbhr @SanthoshVasabhaktula @vrayulu  @vinukumar-vs @Krishnaj20 
@pallakartheekreddy @rhwarrier @anandp504 @reshmi-nair @surendrasinghs @maheshkumargangula
@madhucr @alok-os

**Comment by parthlawate** [comment]
from a long term roadmap perspective will content authoring /workspace, will we move all that to coKreat or will it stay in Sunbird ED ? As while we dont have any coKreat adopters as yet , creation in general is used widely..  Would like to know about this so we can decide if we can add value..

**Reply by kameshbhr** [reply]
@parthlawate Very important question. I think we should deliberate this together - Sunbird ED and coKreat working groups.

**Comment by kameshbhr** [comment]
I nominate to be part of the working group.

**Comment by abhay-samagra** [comment]
Please consider myself and Omkar from Samagra for the working group.
